src/java/org/apache/cassandra/service/accord/AccordJournal.java

```diff
@@ -625,12 +630,18 @@ private static int msVersion(int version)
      * Handling topology changes / epoch shift
      */
 
-    private final class DelayedRequestProcessor extends Thread
+    private class DelayedRequestProcessor implements Interruptible.Task
     {
         private final ManyToOneConcurrentLinkedQueue<RequestContext> delayedRequests = new ManyToOneConcurrentLinkedQueue<>();
         private final LongArrayList waitForEpochs = new LongArrayList();
         private final Long2ObjectHashMap<List<RequestContext>> byEpoch = new Long2ObjectHashMap<>();
         private final AtomicReference<Condition> signal = new AtomicReference<>(Condition.newOneTimeCondition());
+        private Interruptible executor;
```

Comment:
`volatile`?  started in one thread and closed in another thread *might not* have visibility.